app/assets/javascripts/components/Columns/NormalColumns.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Column from './ColumnItem';
+import Stories from "../stories/Stories";
+import Sprints from "../stories/Sprints";
+
+const NormalColumns = ({
```

Comment:
Maybe we should create a single component like a `Columns` component, then pass as props the order of columns, u can pass `firstColumn`, `secondColumn` and `thirdColumn` then based on the `reversed` value, you decided what to pass in each one of them. You can add a children property to `done`, `chillyBin` and `backlog` then in `Columns` you just render that. see the example bellow:

`Columns`
```js
<span className="Columns">
    <Column title={firstColumn.title}
      renderAction={firstColumn.renderAction}
    >
      {firstColumn.children}
    </Column>

    <Column
      title={secondColumn.title}
      renderAction={secondColumn.renderAction}
    >
      {secondColumn.children}
    </Column>

    <Column
      title={thirdColumn.title}
    >
     {thirdColumn.children}
    </Column>
  </span>
``` 
Does it makes senses? what do you think?

Reply:
Nice! I like of idea! I will do this, thanks man.